qt/qssl_wolf/tst_wolfssl.cpp

```diff
@@ -0,0 +1,3676 @@
+    skip_waitForEncrypted                   = false;
+    skip_waitForEncryptedMinusOne           = false;
+    skip_addDefaultCaCertificate            = false;
+    skip_defaultCiphers                     = false;
+    skip_defaultCaCertificates              = false;
+    skip_systemCaCertificates               = false;
+    skip_supportedCiphers                   = false;
+    skip_wildcardCertificateNames           = false;
+    skip_isMatchingHostname                 = false;
+    skip_setEmptyKey                        = false;
+    skip_spontaneousWrite                   = false;
+    skip_setReadBufferSize                  = false;
+    skip_waitForMinusOne                    = false;
+    skip_verifyMode                         = false;
+    skip_qtbug18498_peek                    = false;
+    skip_qtbug18498_peek2                   = false;
+    skip_ignoreSslErrorsListWithSlot        = false;
+    skip_abortOnSslErrors                   = false;
+    skip_ignoreSslErrorsList                = false;
+    skip_resetProxy                         = false;
+    skip_readFromClosedSocket               = false;
+    skip_writeBigChunk                      = false;
+    skip_blacklistedCertificates            = false;
+    skip_sslOptions                         = false;
+    skip_versionAccessors                   = false;
+    skip_encryptWithoutConnecting           = false;
+    skip_resume                             = false;
+    skip_ephemeralServerKey                 = false;
+    skip_signatureAlgorithm                 = false;
+    skip_disabledProtocols                  = false;
+    skip_oldErrorsOnSocketReuse             = false;
+    skip_disconnectFromHostWhenConnected    = false;
+    skip_disconnectFromHostWhenConnecting   = false;
+    
+    skip_ecdhServer                         = false;
+    skip_verifyClientCertificate            = false;
+    skip_dhServer                           = false;
+    skip_readBufferMaxSize                  = false;
+    skip_setEmptyDefaultConfiguration       = false;
+    skip_pskServer                          = false;
+}
+
+QSslSocketPtr tst_QSslWolfSSL::newSocket()
+{
+    const auto socket = QSslSocketPtr::create();
+
+    proxyAuthCalled = 0;
+    connect(socket.data(), SIGNAL(proxyAuthenticationRequired(QNetworkProxy,QAuthenticator*)),
+            SLOT(proxyAuthenticationRequired(QNetworkProxy,QAuthenticator*)),
+            Qt::DirectConnection);
+
+    return socket;
+}
+
+/* functions from QSslSocket to know what they do */
+bool tst_QSslWolfSSL::isMatchingHostname(const QSslCertificate &cert, const QString &peerName)
+{
+    QHostAddress hostAddress(peerName);
+    if (!hostAddress.isNull()) {
+        const auto subjectAlternativeNames = cert.subjectAlternativeNames();
+        const auto ipAddresses = subjectAlternativeNames.equal_range(QSsl::AlternativeNameEntryType::IpAddressEntry);
+        //qDebug() << "subjectAlternativeNames " << subjectAlternativeNames;
```

Comment:
Do we want to uncomment this debug line?  Looks like we use qDebug() below for ipAddress, commonName, etc.

Reply:
Yes. Uncommented. Thanks.